server/requirements_rocm.txt

```diff
@@ -0,0 +1,2 @@
+# We use nightly
+torch>2.1.0 ; python_version >= "3.9" and python_version < "3.13"
```

Comment:
Can you use `pyproject.toml`  to generate those and update `make export-requirements` ?

Reply:
Should I add a `rocm` and `cuda` entry in `tool.poetry.extras`? And update `export-requirements` target to `poetry export -o requirements.txt -E bnb -E quantize -E cuda --without-hashes`?

By the way, how is the target `export-requirements` used?  I can't find where it is called from.